src/oas3/transformers/__tests__/content.test.ts

```diff
@@ -207,6 +211,53 @@ describe('translateMediaTypeObject', () => {
       );
       expect(translatedObject.schema).toHaveProperty('deprecated', true);
     });
+
+    test('will keep the writeOnly property', () => {
```

Comment:
You can group all these tests in a `describe` block so you can transform the object *once* and then do the various `it/expect` statements

Reply:
Done!